news/2025/2025-04-08-beatmap-spotlights-revival-spring-2025.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: post
+title: "Beatmap Spotlights Revival: Spring 2025"
+date: 2025-04-09 01:00:00 +0000
+---
+
+Haven't heard this name for a while? Check out all the major changes to Beatmap Spotlights! This time, we are also seeking fresh blood to ensure the project can run smoothly for many seasons to come!
+
+![](/wiki/shared/news/2025-04-08-beatmap-spotlights-revival-spring-2025/banner.jpg)
+
+It's been some time since we last met here indeed... let's examine what happened in the meantime, since there's certainly a lot to cover.
+
+Even though the playlists are already up and running, we invite you to read on for the story about the new Beatmap Spotlights infrastructure, and the path that led us back to where we are today!
+
+## A lengthy history lesson
+
+Heading into 2024, right after the conclusion of the Autumn 2023 season, the Beatmap Spotlights project was stopped for time being, due to an osu!(lazer) scoring rollout and infrastructure changes related to that.
+
+Some time passed, and alongside the introduction of [Daily Challenge](https://osu.ppy.sh/home/news/2024-07-25-introducing-daily-challenges) in July 2024, discussions began about how they would coexist in the future. After some deliberation, it was decided that both projects would focus on different matters, and aim for a varying player base.
```

Comment:
```suggestion
Some time passed, and alongside the introduction of [Daily Challenge](https://osu.ppy.sh/home/news/2024-07-25-introducing-daily-challenges) in July 2024, discussions began about how that and Beatmap Spotlights would coexist in the future. After some deliberation, it was decided that both projects would focus on different matters, and aim for a varying player base.
```
spotlights isn't mentioned in the paragraph, so "they" is weird

Reply:
i also don't know what "varying player base" means here. if "varying" supposed to be a different word for "different", it doesn't fit lol